src/index.ts

```diff
@@ -15,3 +19,23 @@ export function convertAxeToSarif(axeResults: Axe.AxeResults): SarifLog {
     const sarifConverter = defaultSarifConverter();
     return sarifConverter.convert(decoratedAxeResults, {});
 }
+
+export function sarifReporter(
+    rawResults: AxeRawResult[],
+    {},
```

Comment:
Seems unclear to me what this variable here is/why it's here.

Talking to @dbjorge about this offline and it seems like this is for having reporter options in the future. Since that is the case, I assume we have it here due to establishing a precedent/contract for the future (if that is not necessary/the case, I would prefer this to be removed) and for that, I'd prefer to have a variable name and more importantly a type for the options (which may be an empty object or even null for the moment).

Similarly, in the test cases you can then have a variable that is named appropriately, even if it is set to null/empty object, making it more readable for people in the future.

Reply:
Yeah, we should probably be explicit about the types of the public API surface. This type is going to be controlled by what axe actually passes us (axe is the thing invoking our reporter in normal usage). https://github.com/dequelabs/axe-core/blob/develop/lib/core/public/run.js#L129 is the point where axe invokes the reporter; it looks like this is going to be the same type (or a subset of the same type) that axe.run accepts for options.